insolar/record/record.go

```diff
@@ -164,6 +164,8 @@ type Request interface {
 	// IsTemporaryUploadCode tells us that that request is temporary hack
 	// for uploading code.
 	IsTemporaryUploadCode() bool
+	// IsMutable returns mutable-state of request
+	IsMutable() bool
```

Comment:
This method is not necessary. Mutable is IncomingRequest property.

Reply:
reverted